src/Brancho/Brancho.php

```diff
@@ -96,10 +91,19 @@ protected function getConfigPath(InputInterface $input): string
     /**
      * @param string $pathToConfig
      *
+     * @throws \RuntimeException
+     *
      * @return array
      */
     protected function loadConfig(string $pathToConfig): array
     {
+        if (!file_exists($pathToConfig)) {
```

Comment:
Wouldn't it be better to move the validation for the configuration into the Config class?